tests/kernel/update_kernel.pm

```diff
@@ -470,13 +475,15 @@ sub run {
         if (!check_var('REMOVE_KGRAFT', '1')) {
             # dependencies for heavy load script
             add_qa_head_repo;
-            zypper_call("in ltp-stable");
+            install_package("ltp-stable", trup_reboot => 1);
 
             # update kgraft patch under heavy load
             update_kgraft($incident_klp_pkg, $repo, $incident_id);
 
+            enter_trup_shell if is_transactional;
             zypper_call("rr qa-head");
             zypper_call("rm ltp-stable");
+            exit_trup_shell if is_transactional;
```

Comment:
It'd be nice to add `uninstall_package()` helper function here. `zypper rr` doesn't need trup_shell.

Reply:
https://progress.opensuse.org/issues/168226